docs/Release-Guide.md

```diff
@@ -127,6 +127,27 @@ Bumping the limits may significantly improve build speed.
 * Publish message in Telegram at <https://t.me/VictoriaMetrics_en> and <https://t.me/VictoriaMetrics_ru1>
 * Publish message in Google Groups at <https://groups.google.com/forum/#!forum/victorametrics-users>
 
+## Operator
+
+The operator repository [https://github.com/VictoriaMetrics/operator/](https://github.com/VictoriaMetrics/operator/)
+
+### Bump the version of images
+
+- Bump `Version` field in [file `internal/config/config.go`](https://github.com/VictoriaMetrics/operator/blob/master/internal/config/config.go#L47) with new release version for:
```

Comment:
should we use a permanent link here, so if there are any changes in config.go we'll still have link pointing to the right line?